samples/net/wpanusb/README.rst

```diff
@@ -0,0 +1,92 @@
+.. _wpanusb-sample:
+
+WPANUSB sample
+##############
+
+Overview
+********
+
+This application exports ieee802154 radio over USB to be used in other
+Operation Systems like Linux. In this scenario Linux SoftMAC driver would
+be used implementing ieee802154 stack inside Linux.
+
+Requirements
+************
+
+- Linux machine host connected via USB to Zephyr WPANUSB enabled MCU
+- WPANUSB kernel driver (in the process of being open sourced)
+- IWPAN host tools at [http://wpan.cakelab.org/releases/](http://wpan.cakelab.org/releases/)
+
+Building and Running
+********************
+
+There are configuration files for various setups in the
+samples/net/wpanusb directory:
+
+- :file:`prj.conf`
+  This is the standard default config.  This can be used by itself for
+  hardware which has native 802.15.4 support.
+
+- :file:`overlay-cc1200.conf`
+  This overlay config enables support for CC1200
+
+- :file:`overlay-cc2520.conf`
+  This overlay config enables support for CC2520
+
+- :file:`overlay-mcr20a.conf`
+  This overlay config enables support for MCR20A
+
+Build the WPANUSB sample like this:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :board: <board to use>
+   :conf: <config file to use>
+   :goals: build
+   :compact:
+
+Example building for the Nordic nRF52840 Development Kit:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :host-os: unix
+   :board: nrf52840_pca10056
+   :goals: run
+   :compact:
+
+Example building for the Quark SE C1000 Devboard with TI CC2520 support:
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/net/wpanusb
+   :host-os: unix
+   :board: quark_se_c1000_devboard
```

Comment:
I don't see this as a listed Zephyr supported board in our list http://docs.zephyrproject.org/boards/x86/index.html

Reply:
Interesting, it was the only board which had settings in place for CC2520.  So I figured it was used for testing.  https://github.com/zephyrproject-rtos/zephyr/tree/master/boards/x86/quark_se_c1000_devboard